api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -51,6 +51,12 @@ type KubeadmConfigSpec struct {
 	// PostKubeadmCommands specifies extra commands to run after kubeadm runs
 	// +optional
 	PostKubeadmCommands []string `json:"postKubeadmCommands,omitempty"`
+	// Users specifies extra users to add
+	// +optional
+	Users []User `json:"users,omitempty"`
```

Comment:
Are we tying or have we already tied this struct too tightly to cloud-init? It might be good if we had a cloud-init substructure to host the cloud-init specific bits. Stuff bash wouldn't care about.

Reply:
I don't think we want to get into supporting different subsets of configuration based on output format, otherwise it'll be a mess for users.

Reply:
ok. i'm not terribly picky so long as they are all optional it will still look ok to a user